polus-montage-plugin/plugin.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
I know that this isn't typical but I'd like to start adding comments within the JSON docs to describe what each section is doing. It'll help new people make them more quickly and add more clarity for anyway (me) trying to look for a specific section in the JSON.

Reply:
JSON does not have the capability for commenting. JSON is designed to be data only.